javarosa/util/schema-gen/src/org/javarosa/engine/XFormEnvironment.java

```diff
@@ -93,8 +94,12 @@ private InstanceInitializationFactory createIIF() {
 
     private EvaluationContext getEC() {
```

Comment:
I tried to clear it up, but I'm not sure how much clearer I made it. If it helps, I'm pretty sure `Mockup`s are no longer used, so we could think about taking them out, but would probably want to verify with @ctsims